<a href="https://colab.research.google.com/github/LamarL26/FTI-QSI-26/blob/main/QSI's_PythonProgramming_Day_1.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

---
# **Python Programming: Part I**
---


### **Description**
This notebook is designed to provide hands-on practice with Python basics to solidify the foundation upon which we will build our machine learning and data science skills in the upcoming program.


<br>

### **Lab Structure**
**Part 1**: [Getting Familiar with Google Colab](#p1)

**Part 2**: [Print Statements](#p2)

**Part 3**: [Creating Variables](#p3)

**Part 4**: [Variables and Data Types](#p4)

**Part 5**: [Arithmetic Operators](#p5)

**Part 6**: [Booleans and Comparison Operators](#p6)


**Part 7**: [Conditionals - `if`](#p7)

**Part 8**: [Conditionals - `else`](#p8)

**Part 9**: [Conditionals - `and` & `or`](#p9)

**Part 10**: [Conditionals - `elif`](#p10)

**Part 11**: [Independent Practice](#p11)
<br>

### **Learning Objectives**
 By the end of this lab, we will:
* Recognize how information is outputted to the screen.
* Recognize that variables are a way to store data.
* Recognize the main data types and how to modify them with basic arithmetic operators.


<br>


### **Resources**
* [Python basics Cheat Sheet](https://docs.google.com/document/d/1P3mae56ecUhbi3zq5eCT9T7HamCGEcd3lqqdZvUZAvU/edit?usp=sharing)

<a name="p1"></a>

---
##**Part 1: Getting Familiar with Google Colab**
---

#### **Problem #1.1**

**Together**, let's change the title of our notebook. Rename the notebook to "Python Pre-Training Session 1 Lab Notebook".

#### **Problem #1.2**

**Together**, let's create a new text cell and type: "This is a text cell."

#### **Problem #1.3**

**Together**, let's create a new code cell and type: "This is a code cell."

#### **Problem #1.4**

**Together**, let's run the code cell below.

In [ ]:
"Hello World!"

'Hello World!'

#### **Problem #1.5**

**Together**, let's run the code cell below.

In [ ]:
print('This line of code will execute first.')

print('Then, the computer will run this line of code.')

print('After, this line of code will run.')

This line of code will execute first.
Then, the computer will run this line of code.
After, this line of code will run.


**Notice** how each line of code is displayed after one another. This because our computer runs the code **sequentially**. Whatever is written first in code will output first.

---

<center>

### **Back to Lab Slides**



<a name="p2"></a>

---
## **Part 2: Print Statements**
---

#### **Problem #2.1**

**Together,** let's create a code cell and type: `print("Hello!")`.

#### **Problem #2.2**


**Together,** let's create a code cell and type: `print("Today I learned about the basics of Python programming!")`.

---

### **Try Problems #2.3-2.6 on your own.**




#### **Problem #2.3**


**Independently,** print your favorite color.


#### **Problem #2.4**


**Independently,** print a sentence that describes what state or country you are from.



#### **Problem #2.5**

**Independently,** print the food you had for breakfast today.


#### **Problem #2.6**

The ability to choose between single quotes and double quotes is useful when trying to print either of these characters. We can use one to print the other.

**The code below is provided for you. Run it to print the statements**

In [ ]:
print("My cat's name is Charlie.")

print('Charlie said "meow".')

My cat's name is Charlie.
Charlie said "meow".


---

<center>

### **Back to Lab Slides**





#### **Problem #2.7**

**Together,** let's create a code cell and print your favorite number.

#### **Problem #2.8**


**Together,** let's create a code cell and print the sum of 33 and -11.

---

### **Try Problems #2.9-2.12 on your own.**




#### **Problem #2.9**


**Independently,** print the number of months in a year.

In [ ]:
print("There are 12 months in a year")

There are 12 months in a year


#### **Problem #2.10**



**Independently,** print the current year.


#### **Problem #2.11**



**Independently,** print today's date in the following format: MMDDYYYY.

In [ ]:
print(6,"/",13,"/",2026, sep = "")

6/13/2026



#### **Problem #2.12**



**Independently,** print the sum of your age plus the age of your best friend.

---

<center>

### **Back to Lab Slides**



<a name="p3"></a>


---
## **Part 3: Creating Variables**
---

####**Problem #3.1**

**Together,** let's create a variable called `ten` to store the number 10.

In [ ]:
ten = 10
print(ten)

10


#### **Problem 3.2**

**Together,** let's create a variable called `introduction` to store the word "Hello".

In [ ]:
introduction = "Hello"
print(introduction)

Hello


In [ ]:
intro='Hello'
print(intro)

Hello


---

#### **Try Problems #3.3-3.4 on your own.**



#### **Problem 3.3**

**Independently,** create a variable called `year` to store the number 2026.

In [ ]:
year = "6/13/2026"
print(year)

6/13/2026


#### **Problem 3.4**

**Independently,** create a variable called `month` to store the word "June".

In [ ]:
month = "June"
print(month)

June


---
<center>

###**Back to Lab Slides**

#### **Problem 3.5**

**Together,** let's determine which variable names are valid or invalid.

```
hello, #Invalid
sampleVariable #Valid
my_password #Valid
variable name #Invalid
True #Invalid
```

---

#### **Try Problem #3.6 on your own.**



#### **Problem 3.6**

The variables in the code below are poorly named.

**Independently,** rename the variables so that they are well named.

In [ ]:
my_name = "Jane"
Straight = "https://the-cs.org"
two_two = 4
PLANT = "Orchid"
prime_number = 17

---

###**Back to Lab Slides**

<a name="p4"></a>

---
## **Part 4: Variables & Data Types**
---

#### **Problem #4.1**


**Together,** let's create a variable called `my_name` and assign it to your first name. Print this variable.

In [ ]:
my_name = "Lamar"
print(my_name)

Lamar



#### **Problem #4.2**

**Together,** let's create and name a variable with your favorite number as an integer. Print that variable.

In [ ]:
fav_number = 33
print(fav_number)

33


---
### **Problem #4.3**

**Together,** let's create and name a variable with your favorite number as a float. Print that variable.

In [ ]:
fav_number = 33.0
print(fav_number)

33.0


---

### **Try Problems #4.4-4.6 on your own.**





#### **Problem #4.4**



**Independently,** look through the given declarations. Which ones are incorrect?

```
Var1 = 7
VAR5 = 8
2var = 92
first_last_name = "james gooding"
Hello = 0.283
variable24 = "my favorite food is"
str = "string"
```



#### **Problem #4.5**


**Independently,** create and a name a variable that stores a list of 3 fruits. Print that variable.



In [ ]:
listexample = ["bananas","watermelon","mango"]
print(listexample)

['bananas', 'watermelon', 'mango']


#### **Problem #4.6**

**Independently,** create two variables to represent your height, one variable for feet called `feet` and one for inches called `inches`.

In [ ]:
feet = 5
inches = 11
print(feet,inches)

5 11


---
<center>

###**Back to Lab Slides**


#### **Problem #4.7**

**Independently,** enter your first name, then enter your last name separately. Output your full name.

In [ ]:
first_name = "Lamar"
last_name = "Logan"
full_name = "Lamar Logan"
print("Hi, my name is " + full_name + " and I would love to do Security Engineering. On the documentation, I would put " + first_name + " as my first name and " + last_name + " as my last name.")

Hi, my name is Lamar Logan and I would love to do Security Engineering. On the documentation, I would put Lamar as my first name and Logan as my last name.


<a name="p5"></a>

---
## **Part 5: Arithmetic Operators**
---

#### **Problem #5.1**


**Together,** let's print the sum of 3 and 2. Take note of the output.

In [ ]:
print(3+2)

5



#### **Problem #5.2**

**Together,** let's print the sum of 7.0 and 2.0. Take note of the output.

In [ ]:
print(7.0+2.0)

9.0


#### **Problem #5.3**


**Together,** let's determine what 23 divided by 2 is. Print the answer. Take note of the output.

In [ ]:
print(23/2)

11.5


#### **Problem #5.4**


**Together,** let's print the product of 24.0 and 25.4. Take note of the output.

In [ ]:
print(24.0*25.4)

609.5999999999999


#### **Problem #5.5**


**Together,** let's create and name a variable `fav_number` with your favorite number as an integer. Multiply the variable by 3 and print the result.

In [ ]:
fav_number = 17
result = 17*3
print(result)

51


#### **Problem #5.6**


**Together,** let's create and name two variables `fav_animal` with your favorite animal and `fav_color`. Print out a sentence using both of these.

In [ ]:
fav_animal = "panda"
fav_color = "blue"
print("My favorite animal is a " + fav_animal + " and my favorite color is " + fav_color)

My favorite animal is a panda and my favorite color is blue


#### **Problem #5.7**


**Together,** let's create and name a variable `age` that stores your age as an integer. Create and name another variable with your birthday as a string. Print out a sentence using both of these.

In [ ]:
age = 22
birthday = "November 3st, 2026"

print("I am " + str(age) + " years old, and my birthday is on " + birthday)

I am 22 years old, and my birthday is on November 3st, 2026


---

### **Try Problems #5.8-5.14 on your own.**



#### **Problem #5.8**

**Independently,** by adding variables, determine how many fruits you have in total if you have 2 bananas and 5 apples.

In [ ]:
bananas = 2
apples = 5

print(bananas + apples)

7


#####**Solution**

In [ ]:
bananas = 2
apples = 5
fruits = bananas + apples
print(fruits)

7


#### **Problem #5.9**


You got a 90 on your biology test, and then answered the extra credit question, which gave you an additional 4 points.

**Independently,** calculate and print your final grade.



In [ ]:
original_grade = 90
bonus = 4
test_score = original_grade + bonus
print(test_score)

94


#####**Solution**

In [ ]:
original_grade = 90
bonus = 4
final_grade = original_grade + bonus
print(final_grade)

94


#### **Problem #5.10**


The length and width of a rectangle are 5 and 8, respectively.

**Independently,** calculate the area of a rectangle using these variables and print the result.

In [ ]:
length = 5
width = 8
area = length*width
print(area)

40


#####**Solution**

In [ ]:
length = 5
width = 8
area = length * width
print(area)

40


#### **Problem #5.11**


**Independently,** create three variables `num1`, `num2`, and `num3` and assign them the values of 5, 24, and 67, respectively. Then calculate the average of these three numbers.

In [ ]:
num1 = 5
num2 = 24
num3 = 67
average = (num1 + num2 + num3 / 3)
print(average)


51.33333333333333


#### **Problem #5.12**


**Independently,** create a variable storing someone's first name and another one for their last name. Concatenate them and print out a greeting using both variables.

In [ ]:
first = "Lamar"
last = "Logan"
print("Hey, my name is " + first + " " + last + " and it is a pleasure to meet you!")

Hey, my name is Lamar Logan and it is a pleasure to meet you!


#### **Problem #5.13**


**Independently,** create a variable storing an employee's first name. Create another one called `employee_id` which is an integer.

This employee's username combines their first name and their id together. Create this username and print it out.

In [ ]:
employee_name = "Michaela"
employee_id = 4890564
username = employee_name + str(employee_id)
print(username)

Michaela4890564


#####**Solution**

In [ ]:
first_name = "Alice"
employee_id = 101

# Convert the integer ID to a string before concatenating
username = first_name + str(employee_id)

print("Generated Username:", username)

#### **Problem #5.14**


**Independently,** write a program that takes a website name (like "google") and concatentates the prefix `www.` to the front and the suffix `.com` to the black to create a website address.

In [ ]:
website = "TerpBridge"
front = "www."
end = ".com"
print(front + website + end)

www.TerpBridge.com


---
<center>


###**Back to Lab Slides**



<a name="p6"></a>

---
## **Part 6: Booleans and Comparison Operators**

---

####**Problem #6.1**

**Together,** let's determine which of these booleans are `True`, and which are `False`.

```
3 + 3 == 6 #True
"hello" != "Hello" #True
4 <= 2 #False
5 > -1 #True
10 >= 10 #True
"10" == 10 #False
```

In [ ]:
"10" == 10
#The true or false statement is dependant on the last concatetation

False

####**Problem #6.2**

**Together,** create a variable called `true_boolean` and assign it to True. Next, create a variable called `false_boolean` and assign it to False.

In [ ]:
true_boolean = True
false_boolean = False

####**Problem #6.3**

**Together,** write an expression that checks if `age` is greater than 18.

In [ ]:
age = 20
age > 18

True

---

###**Try Problems #6.3-6.5 on your own.**

#### **Problem #6.3**

**Independently,** write an expression what will check if `test_score` is greater than or equal to 80.

In [ ]:
test_score = 80
test_score >= 80

True

####**Problem 6.4**

You are given the variable `balance` which represents the amount of money in your bank account and `withdraw_amount` which represents the amount of money you are trying to withdraw.

**Independently,** create a boolean variable called `can_withdraw` using boolean operators to determine whether or not you have enough money to complete the withdraw.

In [ ]:
balance = 150
withdraw_amount = 200
can_withdraw = 150 >= 200
can_withdraw

False

####**Problem #6.5**

Given,
```
x = 10
y = 5
```

**Independently,** predict which of these statements will return True, and which will return False.

```
x - y == y #True
y + y == x #True
x == y #False
x != y #True
x > y #True
x <= y #False
```

<a name="p7"></a>

---
## **Part 7: Conditionals - `if`**

---


#### **Problem #7.1**

**Together,** let's create a variable called `temp`. If the temperature is below 40 °F, print a message stating that it is cold outside.

In [ ]:
temp = 35
if temp < 40:
  print("It is frezzing outside!")

It is frezzing outside!


#### **Problem #7.2**

**Together,** let's write some code that prints a New Year's message. Use an if statement to determine whether `year` is a leap year. If it is, then your message should also include this. Test your code on multiple different years.

```
Happy New Year! Good luck in 2024!
2024 is a leap year.
```

**Hint:** 2020 and 2024 are both leap years.

In [ ]:
year = 2024

print("Happy New Year! Good luck in " + str(year) + "!")

if year % 4 == 0:
  print(year, "is a leap year.")

Happy New Year! Good luck in 2024!
2024 is a leap year.


#### **Problem #7.3**

**Together,** let's create a variable called `my_num` and print the absolute value of `my_num`, in other words, if `my_num` is negative you need to make it positive before printing.

In [ ]:
my_num = -5

if my_num < 0:
  my_num + my_num * -1

print("The number is", my_num)

The number is -5


---

### **Try Problems #7.4-7.5 on your own.**



#### **Problem #7.4**

Water only boils at or above 100 °C.

**Independently,** create a variable, `water_temp`, which represents the temperature of the water, and print the statement "The water is boiling" only if the water is boiling.

In [ ]:
water_temp = 105

if water_temp >= 100:
  print("The water is boiling.")

The water is boiling.


#### **Problem #7.5**

**Independently,** write an `if` statement that prints "The number [num] is even" if `num` is even.

In [ ]:
num = 4

if num % 2 == 0:
  print("The number", num, "is even.")

The number 4 is even.


---
<center>

###**Back to Lab Slides**

<a name="p8"></a>

---
## **Part 8: Conditionals - `else`**

---

#### **Problem #8.1**

**Together,** let's build off of your solution from **Problem #7.1** and add an `else` statement.

In [ ]:
temp = 32

if temp > 40:
  print("Go outside. Have fun!")
else:
  print("You better sit down inside!")

You better sit down inside!


#### **Problem #8.2**

**Together,** let's build off of your solution from **Problem #7.2** and add an `else` statement that tells us when the `year` is not a leap year.

In [ ]:
year = 2026

print("Happy New Year! Good luck in " + str(year) + "!")

if year % 4 ==0:
  print("Tt is a leap year.")
else:
  print("It is not a leap year.")

Happy New Year! Good luck in 2026!
It is not a leap year.


---

### **Try Problems #8.3-8.5 on your own.**



#### **Problem #8.3**

**Independently,** build off of your solution from **Problem #7.4** and add an `else` statement.

In [ ]:
water_temp = 95

if water_temp >= 100:
  print("The water is boiling.")
else:
  print("The water is no where near done. Make sure that you keep it on for a little longer.")

The water is no where near done. Make sure that you keep it on for a little longer.


#### **Problem #8.4**

**Independently,** build off of your solution from **Problem #7.5** and modify it so that it prints whether `num` is even or odd.

In [ ]:
num = 3

if num % 2 == 0:
  print("The number", num, "is even.")
else:
  print("The number", num, "is odd.")

The number 3 is odd.


#### **Problem #8.5**

**Independently,** write a program that converts time (hours and minutes) from 24-hr to 12-hr format and displays it.

For example:
```python
hours = 14
minutes = 23
```
Should output:
```
14:23 --> 2:23 PM
```

In [ ]:
hours = 14
minutes = 23

if hours <= 12:
  print(hours, ":", minutes," --> ",hours, ":", minutes, " AM", sep = "")
else:
  new_hours = hours-12
  print(hours, ":", minutes," --> ",new_hours, ":", minutes, " PM", sep = "")


14:23 --> 2:23 PM


---
<center>

###**Back to Lab Slides**

<a name="p9"></a>

---
## **Part 9: Conditionals - `and` & `or`**
---

#### **Problem #9.1**

A robot needs to sort through a shipment of assorted fruits and vegetables and discard any that are either bruised ***or*** moldy.

**Together,** let's finish programming the robot to complete the given task.


In [ ]:
bruised = False
moldy = True

if bruised == True or moldy == True:
  print('Discard this item.')

Discard this item.



#### **Problem #9.2**

The same robot must now properly sort the remaining fruits and vegetables but is only able to identify the different produce based on their `shape` and `color`.


| Produce     | Shape | Color  |
| ----------- | ----- | ------ |
| **apple**   | round | red    |
| **banana**  | long  | yellow |
| **carrot**  | long  | orange |
| **lemon**   | round | yellow |
| **orange**  | round | yellow |


**Together,** let's finish programming the robot to identify the produce from the different combinations of characteristics.

In [ ]:
shape = 'round'
color = 'green'

if shape == 'round' and color == 'red':
  print('Apple')

if shape == 'long' and color == 'yellow':
  print('Banana')

if shape == 'long' and color == 'orange':
  print('Carrot')

if shape == 'round' and color == 'yellow':
  print('Lemon')

if shape == 'round' and color == 'orange':
  print('Orange')

if shape == 'round' and color == 'green':
  print('watermelon')

watermelon


#### **Problem #9.3**

**Together,** let's write a function such that if the `weather` is `raining` or the `temperature` is below 60 °F, it will ouptut "Better wear a jacket!". Otherwise, it will output "No jacket needed."

In [ ]:
weather = "raining"
temperature = 50

if weather == "raining" or temperature < 60:
  print("Better wear a jacket!")
else:
  print("No jacket needed.")

Better wear a jacket!


---

### **Try Problems #9.4-9.5 on your own.**




#### **Problem #9.4**
**Independently,** write a function such that if `cloudy = False` and `raining = True`, it will print "There's a rainbow!". Otherwise, it will print "No rainbow."

In [ ]:
cloudy = False
raining = False

if cloudy == False and raining == True:
  print("There's is a rainbow!")
else:
  print("No rainbow, sadly!")

No rainbow, sadly!


#### **Problem #9.5**


Miles wants to go play a new video game at Peter's house, but Miles' parents will only let him if he has either:
- Finished all of his homework.

&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;**OR**
- Washed the dishes **AND** raked the yard.

**Independently,** replicate this scenario in the code below.

In [ ]:
homework_done = 'Not Done'
dishes_washed = 'Done'
yard_raked = 'Not Done'

if homework_done == 'Done' or  dishes_washed == 'Done' and yard_raked == 'Done':
  print('Well done! Now go and have fun!')
else:
  print('Get back to work!')

Get back to work!


---

###**Back to Lab Slides**

<a name="p10"></a>

---
## **Part 10: Conditionals - `elif`**

---

#### **Problem #10.1**

**Together,** let's build off of your solution from **Problem #8.4** and modify it to print whether a float is either even, odd, or not a whole number.

In [ ]:
num = 13
if num % 2 == 0:
  print('The number', num, 'is even.')
elif num % 4 == 1: #elif statement added
  print('The number', num, 'is odd.')
else:
  print(num, 'is not a whole number.')

The number 13 is odd.


#### **Problem #10.2**

A movie theater sells tickets at different prices based on the following age groups:

- Child (0-12) : \$5
- Teenager (13-19) : \$8
- Adult (20-64) : \$10
- Senior (65 and above) : \$6

When ordering tickets online, customers input an age and the site tells them what age group and the price of ticket are.

**Together,** let's implement this senario with code.

In [ ]:
age = 15

if age <= 12:
  print("You have selected a Child ticket for $5")
elif age <= 14:
  print("You have selected a Teenager ticket for $8")
elif age <= 64:
  print("You have selected an Adult ticket for $10")
else:
  print("You have selected a Senior ticket for $6")

You have selected an Adult ticket for $10


---

### **Try Problems #10.3-10.5 on your own.**




#### **Problem #10.3**

**Independently,** write a program that classifies animals as carnivores, omnivores, or herbivores based on their diet.


In [ ]:
meat = False     # @param {type:"boolean"}
plants = True   # @param {type:"boolean"}

if meat == True and plants == True:
  print("This is an omnivore.")
elif meat == True and plants == False:
  print('This is a straight-up carnivore!')
elif meat == False and plants == True:
  print ('This is a herbivore that only eats plants.')
else:
  print('Surely this animal is dead or served in a kitchen!')


This is a herbivore that only eats plants.


#### **Problem #10.4**

**Independently**, write a function such that if a variable, `num` is more than zero, it will print "num is positive." If `num` is less than zero, it will print "num is negative." Otherwise, it will print "num is zero."

In [ ]:
num = 0

if num > 0:
  print('num is positive.')
elif num< 0:
  print('num is negative.')
else:
  print('num is zero.')

num is zero.


#### **Problem #10.5**

**Independently,** create a function such that it takes in two variables, `color_1` and `color_2` and outputs the result of mixing two primary colors (red, yellow, and blue).

In [ ]:
color_1 = "yellow" # @param ["red", "yellow", 'blue']
color_2 = "blue" # @param ["red", "yellow", 'blue']

if color_1 == "red" and color_2 == "yellow":
  print("orange")
elif color_1 == "blue" and color_2 == "red":
  print("purple")
elif color_1 == "yellow" and color_2 == "blue":
  print("green")
else:
  print("Jesus is Lord!")


#COMPLETE

green


---
### **Back To Lab Slides**


#### **Problem #10.6**

**The code below is provided for you. Run it to simulate the decision to play tennis.Use the drop down menus on the right to adjust the weather conditions.**

In [ ]:
outlook = "sunny"     # @param ["sunny", "overcast", "rain"]
humidity = "high"     # @param ["high", "normal"]
wind = "strong"       # @param ["strong", "weak"]
temperature = "hot"   # @param ["hot", "cold", "warm"]

if outlook == "overcast" or (outlook == "sunny" and humidity == "normal") or (outlook == "rain" and wind == "weak"):
    print("Let's play tennis.")
elif temperature == "hot":
    print("Let's go to the beach.")
else:
    print("Let's do something else.")


Let's go to the beach.


<a name="p11"></a>

---
## **Part 11: Independent Practice**
---

#### **Problem #11.1**


**Independently,** answer the following question.

What's wrong with the following declaration?

A)
```
1var = 323.0 #using a integer with the variable
```
B)
```
month@month = "July" #the @ symbol
```

C)
```
month year = "july2020"
```

#### **Problem #11.2**


**Independently,** create a list named `nums` with values 1, 2, and 3.

In [ ]:
nums = [1,2,3]
print(nums)

[1, 2, 3]


#### **Problem #11.3**


**Independently,** divide 13 by 7. Make sure the answer has decimals.

In [ ]:
13 / 7


1.8571428571428572

#### **Problem #11.4**


**Independently,** reassign the variable, `candybar`, to "Milkyway." Print `candybar` to verify that your reassignment worked.

In [ ]:
candybar = "Snickers"
candybar = "Milkyway"
print(candybar)

Milkyway


#### **Problem #11.5**


**Independently,** answer the following questions.

Using the code below, what will print when you `print(bananas)`? Why?

```
bananas = 4
apples = 3
oranges = banana
bananas = 8
print(bananas)

#As a result, 8 will print.
```

In [ ]:
bananas = 4
apples = 3
oranges = bananas
bananas = 8
print(bananas)

8


#### **Problem #11.6**


**Independently,** name a variable `name` and assign it your name. Then change the value to your instructor's name.

#### **Problem #11.7**


**Independently,** create a list named 'grocery_list' and add 5 items. Print the list.

In [ ]:
grocery_list = ["Fish", "Apple", "Yogurt", "Drinks", "Alspice"]
print(grocery_list)

['Fish', 'Apple', 'Yogurt', 'Drinks', 'Alspice']


#### **Problem #11.8**

**Independently,** answer the following question.

What are the following data types?

```
4323 integer

"word" string

[4.0, 25.4, 192.0] list

["apples", "bananas", "oranges"] list

-1.231 float
```

#### **Problem #11.9**


**Independently,** create a list called `capitals` and list 5 U.S. State capitals. Print the list.

#### **Problem #11.10**


Your plant grew 5.3 inches in 10 days.

**Independently,** create a variable `height` and assign it to value 5.3. Create a variable `days` and assign it to value 10.

Assuming your plant grew the same amount everyday, how much did it grow each day?

In [ ]:
height = 5.3
days = 10
print(height/days)

0.53


**[Answer Here]**

#### **Problem #11.11**


Jake has 5 bags of coins. Each bag has 18 coins in it.

**Independently,** create variables `bags` and `coins` and print their product: `total`.

How many coins in total does Jake have?


In [ ]:
bags = 5
coins = 18
total = bags * coins
print(total)

90


**[Answer Here]**

---
#End of notebook

© 2025 The Coding School, All rights reserved